changelog.d/9966.feature

```diff
@@ -0,0 +1 @@
+Support [MSC1772](https://github.com/matrix-org/matrix-doc/pull/1772).
```

Comment:
I think we should be a bit more explicit about what this actually means for the average synapse sysadmin